### DISCUSSION TITLE
Sury does not have a Release file

### DISCUSSION THREAD

**Discussion by n0valis** [oerdnj/deb.sury.org#2000] [General]
I am on LMDE5 Kernel 6.10, so it's Bullseye 11.2
I am still relatively new to Linux so please forgive my ignorance.
I can't get to the bootom of this error message.
I have added gpg key and sury repo to apt list, but get the following for apt update (resp. curl -sSL https://packages.sury.org/php/README.txt | sudo bash -x)
E: The repository 'https://packages.sury.org/php stretch Release' does no longer have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.

I read that this happens to EOL systems but that is obviusly not the case here.
Why is this happening? And how can I fix this?
I'd like to install php8 to get nextcloud running.

Thanks

**Comment by bashgeek** [comment]
You're trying to install from the Debian stretch repo - which obviously is EOL for a while now. If you're really running bullseye, you might want to change your sources file accordingly

**Comment by oerdnj** [comment]
Obviously, the place where the script checks for codename still says „stretch”.

Linux Mint is not supported. That’s explicitly stated in the FAQ, so you are own your own.

**Comment by n0valis** [comment]
Sorry copy and paste error 
It actually says 
E: The repository 'https://packages.sury.org/php **elsie** Release' does no longer have a Release file.
